rskj-core/src/main/java/co/rsk/net/sync/BaseSyncState.java

```diff
@@ -44,7 +47,7 @@ public void newBlockHeaders(List<BlockHeader> chunk) {
     }
 
     @Override
-    public void newBody(BodyResponseMessage message) {
+    public void newBody(BodyResponseMessage message, MessageChannel peer) {
         // TODO(mc) do peer scoring, banning and logging
```

Comment:
This will remain like this until more tests on the network. we want to be sure that we are not being too rigid with old messages from peers that are slow or have bad connections.